zmon-controller-app/src/main/java/org/zalando/zmon/api/domain/EntityFilterResponse.java

```diff
@@ -32,4 +37,5 @@ public SimpleEntity(String id, String type) {
 
     public int count = 0;
     public final Collection<SimpleEntity> entities = new ArrayList<>(25);
+    public String message = new String("");
```

Comment:
This is almost the same as
```java
public String message = "";
```
But actually happens to be worse due to explicit heap allocations. Check this [article](http://www.java67.com/2014/08/difference-between-string-literal-and-new-String-object-Java.html).

Reply:
Fixed